### PR TITLE
Integration of Firebase Cloud Messaging (FCM)

### PR DESCRIPTION
**Pull request by santhoshvaddi**
- Updated index.js with Push Notification Method.

### CODE REVIEW

**Comment by fhinkel** [comment]
Running the linter again. If it fails, please fix with `npm run lint -- --fix`.

File: functions/dialogflow/functions/index.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ const db = admin.firestore();
 
 // [START save_token_to_firebase]
 function saveOAuthToken(context, oauthToken) {
-  const docRef = db.collection('DialogflowTokens').doc('OauthToken');
+  const docRef = db.collection('ShotLivedAuthTokens').doc('OauthToken');
```
Comment (nnegrey) [inline functions/dialogflow/functions/index.js]: `ShortLivedAuthTokens`

**Comment by nnegrey** [comment]
Might be worth taking the time to update the style form this PRs update to the STS sample. 
https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/1384

File: functions/dialogflow/functions/index.js (hunk 1 above)
Comment (santhoshvaddi) [reply functions/dialogflow/functions/index.js]: Done.

**Pull request merged**